lib/vc8000nanoe/inc/enccommon.h

```diff
@@ -153,8 +153,12 @@ typedef struct
 
 /* General tools */
 #define ABS(x)          ((x) < (0) ? -(x) : (x))
+#ifndef MAX
 #define MAX(a, b)       ((a) > (b) ?  (a) : (b))
+#endif
+#ifndef MIN
 #define MIN(a, b)       ((a) < (b) ?  (a) : (b))
+#endif
```

Comment:
Use of #ifndef is forbidden. See https://docs.zephyrproject.org/latest/contribute/coding_guidelines/index.html#rule-a-3-macro-name-collisions

Reply:
To workaround this, provide and use an alternate definition:
```diff
-#define MAX(a, b)       ((a) > (b) ?  (a) : (b))
+#define STM32_MAX(a, b)       ((a) > (b) ?  (a) : (b))
```